dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -2283,16 +2284,23 @@ RtpsUdpDataLink::bundle_mapped_meta_submessages(AddrDestMetaSubmessageMap& adr_m
           }
         }
         // Attempt to add the submessage meta_submessage to the bundle
-        bool result = false;
+        bool result = false, unique = false;
+        ACE_UNUSED_ARG(unique);
```

Comment:
Why is this needed?

Reply:
I assumed that if we were setting unique below but never referencing it when the asserts were optimized out, some compilers would complain it was unused / unreferenced.

Reply:
I missed the detail that this was assigned-to but not read-from.